### PR TITLE
feature/190 worked on home screen responsive

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/demo/assets/tonomy-mobile-logo.png
Comment (theblockstalk) [inline src/demo/assets/tonomy-mobile-logo.png:1]: why not use the existing image in Tonomy-App-Websites/public/tonomy-logo1024.png ?

File: src/demo/pages/Home.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 body {
-  background: linear-gradient(115.56deg, #C1EEF8 44.33%, #67D7ED 89.31%) !important;
```
Comment (theblockstalk) [inline src/demo/pages/Home.css:2]: these should use the theme colours. Also not use !important

File: src/demo/assets/tonomy-mobile-logo.png
Comment (sadiabbasi) [reply src/demo/assets/tonomy-mobile-logo.png:1]: because when i increase size it looks blur... so i downloaded which they use in figma mobile view

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**